nephio-ansible-install/README.md

```diff
@@ -56,6 +57,8 @@ all:
     host_min_root_disk_space: 50 # minimum required disk space before install; value in GB
     tmp_directory: "/tmp"
     bin_directory: "/usr/local/bin"
+    installer_lib: <provide full path for installer_lib direcotry under nephio-ansible-install-offline>
```

Comment:
what would the value for this field look like? Is nephio-ansible-install-offline a directory?

Reply:
For using the offline installation, we just need to set installation_mode property in nephio.yaml to offline.